test/IRGen/raw_layout.swift

```diff
@@ -64,18 +64,75 @@ struct Keymaster: ~Copyable {
     let lock3: Lock
 }
 
-/*
-TODO: Dependent layout not yet implemented
+// Dependent Layouts:
+// These types have their layouts all zeroed out and will be fixed up at
+// runtime.
 
+// CHECK-LABEL: @"$s{{[A-Za-z0-9_]*}}4CellVWV" = {{.*}} %swift.vwtable
+// size
+// CHECK-SAME:  , {{i64|i32}} 0
+// stride
+// CHECK-SAME:  , {{i64|i32}} 0
+// flags: alignment 0, incomplete
+// CHECK-SAME:  , <i32 0x400000>
 @_rawLayout(like: T)
 struct Cell<T>: ~Copyable {}
 
+// CHECK-LABEL: @"$s{{[A-Za-z0-9_]*}}10PaddedCellVWV" = {{.*}} %swift.vwtable
+// size
+// CHECK-SAME:  , {{i64|i32}} 0
+// stride
+// CHECK-SAME:  , {{i64|i32}} 0
+// flags: alignment 0, incomplete
+// CHECK-SAME:  , <i32 0x400000>
 @_rawLayout(likeArrayOf: T, count: 1)
 struct PaddedCell<T>: ~Copyable {}
 
+// CHECK-LABEL: @"$s{{[A-Za-z0-9_]*}}14SmallVectorBufVWV" = {{.*}} %swift.vwtable
+// size
+// CHECK-SAME:  , {{i64|i32}} 0
+// stride
+// CHECK-SAME:  , {{i64|i32}} 0
+// flags: alignment 0, incomplete
+// CHECK-SAME:  , <i32 0x400000>
 @_rawLayout(likeArrayOf: T, count: 8)
 struct SmallVectorBuf<T>: ~Copyable {}
-*/
+
+// CHECK-LABEL: @"$s{{[A-Za-z0-9_]*}}8UsesCellVWV" = {{.*}} %swift.vwtable
+// size
+// CHECK-SAME:  , {{i64|i32}} 16
+// stride
+// CHECK-SAME:  , {{i64|i32}} 16
+// flags: alignment 7, noncopyable
+// CHECK-SAME:  , <i32 0x800007>
+struct UsesCell: ~Copyable {
+    let regularInt: Int
+    let specialInt: Cell<Int>
```

Comment:
`Int`'s size is platform-dependent, which means the IR output will be platform dependent. Using `Int32` here might help avoid that, otherwise you'll need separate `CHECK-64-SAME` and `CHECK-32-SAME` lines.